feature/movie/src/main/java/com/chan/movie/ui/main/MovieSearchViewModel.kt

```diff
@@ -132,7 +142,7 @@ class MovieSearchViewModel @Inject constructor(
     }
 
     companion object {
-        private const val INTERVAL_KEYWORD_SEARCH = 1500L
+        private const val INTERVAL_KEYWORD_SEARCH = 1000L
```

Comment:
```suggestion
        private const val INTERVAL_KEYWORD_SEARCH = 1_000L
```
P3. 언더바를 사용하면 가독성이 좋아져요

Reply:
04beed279cccc4090dde733691ed2a20360a2f3f 이런 기본을 놓치다니!!🤣 피드백 감사합니다.